src/core/EdFi.DataManagementService.Core/ApiService.cs

```diff
@@ -264,4 +264,104 @@ public IList<IDataModelInfo> GetDataModelInfo()
         }
         return result;
     }
+
+    /// <summary>
+    /// Get resource dependencies
+    /// </summary>
+    /// <returns>JSON array ordered by dependency sequence</returns>
+    public JsonArray GetDependencies()
+    {
+        JsonNode schema = _apiSchemaProvider.ApiSchemaRootNode;
+
+        KeyValuePair<string, JsonNode?>[]? projectSchemas = schema["projectSchemas"]?.AsObject().ToArray();
+        if (projectSchemas == null || projectSchemas.Length == 0)
+        {
+            string errorMessage = "No projectSchemas found, ApiSchema.json is invalid";
+            _logger.LogCritical(errorMessage);
+            throw new InvalidOperationException(errorMessage);
+        }
+
+        List<JsonNode> projectSchemaNodes = projectSchemas
+            .Where(x => x.Value != null)
+            .Select(x => x.Value ?? new JsonObject())
+            .ToList();
+
+        var dependenciesJsonArray = new JsonArray();
+        foreach (JsonNode projectSchemaNode in projectSchemaNodes)
```

Comment:
I'd move everything above here into `ApiSchemaDocument`, into a method `List<JsonNode> GetAllProjectSchemaNodes()`. `ApiSchemaDocument` already does this for single project schema nodes.